lightning/tidb-lightning.toml

```diff
@@ -89,26 +89,24 @@ driver = "file"
 # - origin. keep the checkpoints data unchanged.
 #keep-after-success = "remove"
 
-
 [conflict]
-# Starting from v7.3.0, a new version of strategy is introduced to handle conflicting data. The default value is "". Starting from v8.0.0, TiDB Lightning optimizes the conflict strategy for both physical and logical import modes (experimental).
```

Comment:
about removing "experimental": is this the final PR to improve performance?

Also please post the test result

Reply:
This is the final PR for 8.1 GA.

We have listed the test plan for 8.1 and 8.2. There may be additional PR for 8.2.

The test result will be posted here when it is done.